### PR TITLE
Add anchors to foldable questions in Newcomers' Guide

### LINKED ISSUE #64: Anchors for folded questions missing on Newcomers' Guide
As reported over Discord, currently there's no anchors on any of the folded questions in the Newcomer's Guide, which prevents users from linking to a question.

### PR DESCRIPTION
**Pull request by pluiedev**
Fixes #64.

### CODE REVIEW

File: src/pages/en/about/newcomer-guide.mdx [hunk 1]
```diff
@@ -1,11 +1,10 @@
 ---
 title: Newcomers' Guide
-description: A newcomers'
+description: A newcomers' guide to Quilt—information, questions and concerns
```
Comment (gdude2002) [inline src/pages/en/about/newcomer-guide.mdx:3]: What's Quilt-information?

File: src/components/atoms/Foldable.astro [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+}
+const { id } = Astro.props as Props;
+---
+
+<Message>
+	<details id={id}>
+		<summary>
+			<i class="summary-icon is-ltr fa-solid fa-lg fa-circle-chevron-right" />
+			<i class="summary-icon is-rtl fa-solid fa-lg fa-circle-chevron-left" />
+			<span><slot name="title" /></span>
```
Comment (gdude2002) [inline src/components/atoms/Foldable.astro:15]: A newline should be placed both above and below the span

**Review by gdude2002 — CHANGES_REQUESTED** [review]

File: src/pages/en/about/newcomer-guide.mdx (hunk 1 above)
Comment (obj-obj) [reply src/pages/en/about/newcomer-guide.mdx:3]: I think it's meant to be Quilt - Information, questions, and answers

Comment (pluiedev) [reply src/pages/en/about/newcomer-guide.mdx:3]: Yeah it's an em dash, haha

**Comment by gdude2002** [comment]
Any time to address the reviews?

**Comment by pluiedev** [comment]
> Any time to address the reviews?

Just did :p

**Comment by gdude2002** [comment]
Nope, you didn't address either of them, haha

**Comment by pluiedev** [comment]
> Nope, you didn't address either of them, haha

I mean... the first one is caused by GitHub displaying an em dash as almost like a hyphen, which obj-obj has already pointed out—[a hyphen there is incorrect anyway](https://en.wikipedia.org/wiki/Dash#Em_dash), and the second one is [fixed in the latest commit.](https://github.com/QuiltMC/quiltmc.org/pull/65/commits/b7c8de69635b19bb81394496d74c982fc01e9e4f)

**Comment by gdude2002** [comment]
> > Nope, you didn't address either of them, haha
> 
> I mean... the first one is caused by GitHub displaying an em dash as almost like a hyphen, which obj-obj has already pointed out—I guess there is [some controversy on which dash is appropriate and when](https://en.wikipedia.org/wiki/Dash#En_dash_versus_em_dash), but really it shouldn't block anything; the second one is [fixed in the latest commit.](https://github.com/QuiltMC/quiltmc.org/pull/65/commits/b7c8de69635b19bb81394496d74c982fc01e9e4f)

There should be a space either side of the dash though, and I asked for a newline either side of the span, not to split the span onto multiple lines, haha

**Comment by pluiedev** [comment]
> > > Nope, you didn't address either of them, haha
> > 
> > 
> > I mean... the first one is caused by GitHub displaying an em dash as almost like a hyphen, which obj-obj has already pointed out—I guess there is [some controversy on which dash is appropriate and when](https://en.wikipedia.org/wiki/Dash#En_dash_versus_em_dash), but really it shouldn't block anything; the second one is [fixed in the latest commit.](https://github.com/QuiltMC/quiltmc.org/pull/65/commits/b7c8de69635b19bb81394496d74c982fc01e9e4f)
> 
> There should be a space either side of the dash though, and I asked for a newline either side of the span, not to split the span onto multiple lines, haha

Ah okay. Sometimes I wonder if I can even read /hj

Should be fixed in the latest commit @gdude2002

**Comment by gdude2002** [comment]
It's not!

image

I'm gonna merge now regardless b/c this is silly.

**Pull request merged**